src/core/messages.js

```diff
@@ -1,27 +1,27 @@
 const messages = [
-  "Nice job",
-  "Looks good",
-  "Clean fix!",
-  "Well done",
-  "Thank you! Really nice PR",
-  "Great job",
-  "Looks amazing",
+  'Nice job',
```

Comment:
ESLint changes `""` to `''`, but Prettier changes to `''`. Prettier runs after ESLint, so the problem will be that when we save file, our quotes will be `''` and the ESLint will have errors